module/module.go

```diff
@@ -868,7 +868,7 @@ func (m *Module) AddStream(ctx context.Context, req *streampb.AddStreamRequest)
 	vcss, ok := m.streamSourceByName[name]
 	if !ok {
 		err := errors.New("unknown stream for resource")
-		m.logger.CWarnw(ctx, err.Error(), "name", name, "streamSourceByName", fmt.Sprintf("%#v", m.streamSourceByName))
+		m.logger.CWarnw(ctx, err.Error(), "name", name.String(), "streamSourceByName", fmt.Sprintf("%#v", m.streamSourceByName))
```

Comment:
Netcode is broadly aware there's an issue with these things. We haven't spent the time to understand what's going on here.

Reply:
I'm not sure this is the same problem @cheukt saw; that might be https://github.com/viamrobotics/rdk/pull/4820 (unless there was another issue you're referring to.) I understand why `name` -> `name.String()` makes sense here, though.

Reply:
Yeah that proto stuff was I think where Cheuk also saw this going wrong. Not sure if that PR addressed this, or if there even was an understanding of this problem yet..